include/oneapi/dpl/pstl/utils.h

```diff
@@ -471,27 +471,9 @@ __pstl_left_bound(_Buffer& __a, _Index __first, _Index __last, const _Value& __v
 using __or_semantic = ::std::true_type;
 using __first_semantic = ::std::false_type;
 
-// Define __void_type via this structure to handle redefinition issue.
-// See CWG 1558 for information about it.
-template <typename... _Ts>
-struct __make_void_type
-{
-    using __type = void;
-};
-
-template <typename... _Ts>
-using __void_type = typename __make_void_type<_Ts...>::__type;
```

Comment:
It looks that `__void_type` is still used here:
https://github.com/oneapi-src/oneDPL/blob/main/include/oneapi/dpl/pstl/iterator_defs.h#L41

Reply:
Fixed